src/helpers/applyPrototypes.js

```diff
@@ -1,11 +1,25 @@
 // Make NW.js and Node globals available in Vue
-export default function applyPrototypes (app) {
-  app.isDesktop = !!window.nw;
+export default function applyPrototypes (component) {
+  console.log(2, { component });
+  if (component && component.config && component.config.globalProperties) {
+    component.config.globalProperties.isDesktop = !!window.nw;
 
-  if (window.nw) {
-    app.nw = window.nw;
-    app.process = window.nw.process;
-    app.require = window.nw.require;
-    app.global = global;
+    if (window.nw) {
+      component.config.globalProperties.nw = window.nw;
+      component.config.globalProperties.process = window.nw.process;
+      component.config.globalProperties.require = window.nw.require;
+      component.config.globalProperties.global = global;
```

Comment:
* `main.js` passes in the `app` into this helper to set up globally accessible variables for all components.
* These variables are then referenced in the components. This works when running the code.
* So when setting up the tests and mounting the components, we need to also give them access to these globals.